steps/release-notes-extractor/0.0.1/step.yml

```diff
@@ -0,0 +1,59 @@
+title: Release notes extractor
+summary: |
+  Easily extract release notes for your project's changelog
+description: |
+  With this step you can easily extract the last entry of your project's changelog to be used as release notes!
+website: https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step
+source_code_url: https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step
+support_url: https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step
+published_at: 2019-11-29T19:52:50.120568739+02:00
+source:
+  git: https://gitlab.com/areebgroup-open-source/bitrise-release-notes-step.git
+  commit: 1894dcf471fca456bcf045eb18d5609ae212e4c3
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- changelog_file_path: CHANGELOG.md
+  opts:
+    description: |
```

Comment:
I think it would make sense to clarify CHANGELOG format requirements if any.

Reply:
Will fix it, thanks.